src/main/java/com/google/sps/servlets/utils/SentimentBucket.java

```diff
@@ -0,0 +1,79 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets.utils;
+
+import java.util.List;
+
+public class SentimentBucket {
+
+  private List<UserComment> topNComments; // userComment with top N magnitude within this interval
+  private Integer frequency; // number of comments whose sentiment score is in this interval
+  private Range intervalRange;
```

Comment:
java nit:

Usually unless you have a need for a primitive to be used as an `Object`, it's better to keep them as a primitive.

There's slightly more cognitive + computational overhead associated w/ objects since they can possibly be `null` and have slightly more memory overhead.

In this case it doesn't really make a huge difference, but it's generally good practice.